content/kotlin/concepts/hashmap/terms/get.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.get()'
+Description: 'The get function is used to retrieve the value associated with a particular key in a HashMap.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'HashMap'
+  - 'Data Types'
+  - 'Collections'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+In Kotlin, the `get()` function is a method of the Map interface that is used to retrieve the value associated with a particular key.
+
+## Syntax
+
+```pseudo
+fun <K, V> Map<K, V>.get(key: K): V?
```

Comment:
```suggestion
myHashMap.get(key: K): V?
```

Not sure why the previous syntax was there; this `.get()` should speak more directly to HashMaps.